### DISCUSSION TITLE
Can anybody give me some advices about my Portfolio?

### DISCUSSION THREAD

**Discussion by elsycharles** [community/community#133936] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

I'm new about Porfolio thing, I created this Porfolio about 4 days ago and It is under development yet but I just want some advices in it.
What should I add, delete, modify, etc.
This is the Portfolio link : [CHARLES Elsy](https://charles-elsy.vercel.app/)
Thanks in advance

**Comment by ShenalSen** [comment]
Hi bro it looks like this
![image](https://github.com/user-attachments/assets/2a2bb9d8-8ab7-41e5-a6f3-cfa3ac5ead7c)
your link is not working

**Reply by elsycharles** [reply]
Sorry, I mistaken something, I've fix it now it works

**Reply by ShenalSen** [reply]
I think its cool , good job man!

**Reply by elsycharles** [reply]
Thanks man

**Comment by NicholasGannon** [comment]
Hey, I really enjoyed the style, animations, and especially the sidebar for navigation on your portfolio!

**Reply by elsycharles** [reply]
Thank you so much for your answer

**Comment by MiguelCardoso19** [comment]
Your portfolio design is neat. Yet I would advise doing a scrollbar in a more elegant way to be in harmony with the entire visual appearance than employing the standard pattern.

**Reply by elsycharles** [reply]
Get it, I will update this soon.
Thanks for the advice, it's really helpful.

**Comment by Lakshan-Madushanka** [comment]
Overall good design. But I would like to point out following pitfalls that you can take steps to make it better.

- Content overflows horizontally. This is the main issue I noticed. The problem stems from mixing fixed positioning with a margin pattern, which caused it.
- Style the scrollbar, particularly the sidebar's scrollbar.
- You can make the sidebar more visually appealing. It currently takes up too much width; consider making it more stylish.
- When scrolling down, the content overflows into the header. You can easily prevent this by using z-index.

**Reply by elsycharles** [reply]
Thank you so much for noticing that problem, and thank you for the solution you gave to resolve this

**Comment by AbuBakkarRaihan** [comment]
Your portfolio looks awesome! you can use those suggestions since it still in progress:

1. Simplicity and Organization – makesure that you can locate everything easily, including the about page, projects as well as your contact details.

2. Show Off Your Best Work – Pinpoint a couple of key projects you have done and explain briefly what specifically you did and the instruments you employed.

3. An Optimal Mobile Viewing Experience – Take a moment to confirm its appearance on the desktop screen and at the same time on the miniature hand held gadgets.

**Reply by elsycharles** [reply]
Thanks for the suggestions,
I'll take note and consider those improvements.

**Reply by AbuBakkarRaihan** [reply]
@elsy-sc  if possible Mark me as a answer [For galaxy brain badge 😉]
